Update Presto functions coverage map

pramodsatya opened the pull request.

mbasmanova reviewed (changes requested):
How did you update these maps? I expect changes to velox/functions/prestosql/coverage/data files, but I don't see these.

I also expect that you use SHOW FUNCTIONS Presto command to get a full up-to-date list of all functions, then update the files above, then run the script to update the maps.

pramodsatya commented:
@mbasmanova, I had updated these maps by following the instructions in: ```velox/functions/prestosql/coverage/README.md```
I also ran ```scripts/gen-docs.sh``` to get the html pages with updated functions list, and verified that these functions are also in ``velox/functions/prestosql/coverage/data/all_aggregate_functions.txt``` and ```velox/functions/prestosql/coverage/data/all_scalar_functions.txt```. 

I have updated the functions list by running ```SHOW FUNCTIONS```, and then updated the coverage map.

mbasmanova commented on velox/functions/prestosql/coverage/data/all_scalar_functions.txt:
```diff
@@ -83,10 +88,14 @@ doy
 enum_key
 exp
 expand_envelope
-features
```
is 'features' no longer available?

```
> show functions like 'features';
 Function |    Return Type     |                                 Argument Types                                 | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+--------------------+--------------------------------------------------------------------------------+---------------+---------------+-------------+----------------+----------+-----------+----------
 features | map(bigint,double) | double                                                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double                                                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double                                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double                                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double, double         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double, double, double | scalar        | true          |             | false          | true     | false     |
(10 rows)
```

pramodsatya replied:
I am getting this output currently:
```
presto:system> show functions like 'features';
 Function | Return Type | Argument Types | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+----------------+---------------+---------------+-------------+----------------+----------+-----------+----------
(0 rows)
```
Will recheck and update, thanks for sharing.

mbasmanova commented on velox/functions/prestosql/coverage/data/all_scalar_functions.txt:
```diff
@@ -207,8 +233,8 @@ regexp_extract_all
 regexp_like
 regexp_replace
 regexp_split
-regress
```
why is this removed?

```
> show functions like 'regress';
 Function | Return Type |        Argument Types         | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+-------------------------------+---------------+---------------+-------------+----------------+----------+-----------+----------
 regress  | double      | map(bigint,double), Regressor | scalar        | true          |             | false          | true     | false     |
(1 row)
```

pramodsatya replied:
Apologies, I did not find it in ```SHOW FUNCTIONS``` output and this is what I see:
```
presto:system> show functions like 'regress';
 Function | Return Type | Argument Types | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+----------------+---------------+---------------+-------------+----------------+----------+-----------+----------
(0 rows)
```
Will recheck and update the PR.

mbasmanova replied:
These are defined in presto-ml/src/main/java/com/facebook/presto/ml/MLFeaturesFunctions.java

pramodsatya replied:
Thanks, yes I found them here, added back ```regress``` and ```features``` and updated the coverage map.

mbasmanova reviewed (commented):
Thanks. Would you, please, update PR description to explain what you have done to arrive at this PR?

mbasmanova reviewed (approved):
Thanks.

aditi-pandit commented:
> Thanks. Would you, please, update PR description to explain what you have done to arrive at this PR?

Lets update the instructions at https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/coverage/README.md as well with the additional step for SHOW FUNCTIONS and getting signatures across the modlues.
